src/main/java/io/mosscloud/primenumbers/PrimeNumbersApp.java

```diff
@@ -1,55 +1,44 @@
-        int maxNumberOfRange;
-
-
-
-//        ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
-        Scanner scanner = new Scanner(System.in);
-        while (true) {
-            System.out.println("Podaj pierwszą liczbę z przedziału");
-            String minNumberOfRangeAsString = scanner.next();
-            try {
-                minNumberOfRange = Integer.parseInt(minNumberOfRangeAsString);
-                if (minNumberOfRange <= 1) {
-                    minNumberOfRange = 2;
-                    System.out.println("Liczby pierwszę są liczbami dodatnimi, a ich przedział zaczyna sie od 2 ;)");
-                    break;
-                }
-            }catch(NumberFormatException numberFormatException){
-                System.out.println("podaj liczbe calkowitą");
-            }
-        }
-
-        System.out.println("Podaj ostatnią liczbę z przedziału");
-        maxNumberOfRange = scanner.nextInt();
+//        Integer minNumberOfRange = null;
+//        Integer maxNumberOfRange = null;
+//
+//
+//
+//        Scanner scanner = new Scanner(System.in);
+//
+//        while (minNumberOfRange == null|| maxNumberOfRange == null) {
+//            System.out.println("Podaj pierwszą liczbę z przedziału");
+//            try {
+//                minNumberOfRange = Integer.parseInt(scanner.next());
+//
+//                if (minNumberOfRange <= 1) {
+//                    minNumberOfRange = 2;
+//                    System.out.println("Liczby pierwszę są liczbami dodatnimi, a ich przedział zaczyna sie od 2 ;)");
+//
+//                }
+//                System.out.println("Podaj ostatnią liczbę z przedziału");
+//                maxNumberOfRange = Integer.parseInt(scanner.next());
+//            }catch(NumberFormatException numberFormatException){
+//                System.out.println("podaj liczbe calkowitą");
+//            }
+//
+//        }
+        GetRangeOfPrimeNumbers getRangeOfPrimeNumbers = new GetRangeOfPrimeNumbers();
+        int minNumberOfRange = getRangeOfPrimeNumbers.getMinNumberOfRange();
+        int maxNumberOfRange = getRangeOfPrimeNumbers.getMaxNumberOfRange();
+        CountPrimeNumbers countPrimeNumbers = new CountPrimeNumbers();
+        countPrimeNumbers.printPrimeNumbers(minNumberOfRange, maxNumberOfRange);
 
 
-        for (int i = minNumberOfRange; i <= maxNumberOfRange; i++) {
-            boolean iIsPrimeNumber = true;
-            int minDividingsNumber = 2;
-            int maxDividingsNumber = i - 1;
-            for (int j = minDividingsNumber; j <= maxDividingsNumber; j++) {
-                if (i % j == 0) {
-                    iIsPrimeNumber = false;
-                    break;
-                }
-            }
-            if (iIsPrimeNumber) {
-                System.out.println(i);
-            }
-        }
     }
 }
 
 //TODO
-//
-//
+// obsluzyc max range number w ten sam sposob ale zeby nie powielac kodu z dobrymi praktykami
```

Comment:
jeżeli to są wymagania "biznesowe" to w opisie zadania powinno to być a nie tu, tez do usunięcia